<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetvalueDialog</class>
 <widget class="QDialog" name="SetvalueDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Update values</string>
  </property>
  <property name="windowIcon">
   <iconset theme="edit-paste">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="addButton">
       <property name="toolTip">
        <string>Add condition.</string>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
       <property name="icon">
        <iconset theme="list-add">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="removeButton">
       <property name="toolTip">
        <string>Remove selected condition.</string>
       </property>
       <property name="text">
        <string>Remove</string>
       </property>
       <property name="icon">
        <iconset theme="list-remove">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="procedButton">
       <property name="text">
        <string>Proced</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QTableWidget" name="Values">
     <property name="toolTip">
      <string>&lt;p&gt;List of additional rules. All selected rules must be met to delete object. You can type &quot;$field_name&quot; to use value of selected field in value column.&lt;/p&gt;</string>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Key</string>
      </property>
      <property name="toolTip">
       <string>Key to check.</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Value</string>
      </property>
      <property name="toolTip">
       <string>Required key value.</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="fieldLabel">
     <property name="text">
      <string>Field</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="Field">
     <property name="toolTip">
      <string>Key to set value.</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="valueLabel">
     <property name="text">
      <string>Value</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QProgressBar" name="progressBar"/>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="Value">
     <property name="toolTip">
      <string>&lt;p&gt;New field data. You can type &quot;$field_name&quot; to use value of selected field.&lt;/p&gt;</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="Class">
     <property name="toolTip">
      <string>&lt;p&gt;Classes of items to update data. Next values can be separated by comma sign. Empty string means all classes.&lt;/p&gt;</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Class</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>addButton</sender>
   <signal>clicked()</signal>
   <receiver>SetvalueDialog</receiver>
   <slot>AddButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>17</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>57</x>
     <y>416</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>removeButton</sender>
   <signal>clicked()</signal>
   <receiver>SetvalueDialog</receiver>
   <slot>RemoveButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>50</x>
     <y>277</y>
    </hint>
    <hint type="destinationlabel">
     <x>50</x>
     <y>402</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>SetvalueDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>285</x>
     <y>275</y>
    </hint>
    <hint type="destinationlabel">
     <x>285</x>
     <y>452</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>procedButton</sender>
   <signal>clicked()</signal>
   <receiver>SetvalueDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>335</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel">
     <x>347</x>
     <y>493</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>RemoveButtonClicked()</slot>
  <slot>AddButtonClicked()</slot>
 </slots>
</ui>
